Do NOT install as 'ldd/ldconfig' since this may
result in this ldd being used when the native/host ldd is wanted.
Fixes a problem when configuring ncurses; uclibc's ldd produces
output which is incompatible with ncurses-5.7 'configure'

NOTE: host-ldd is NOT a cross-ldd (uclibc-0.9.33.2) but assumes
      target ELF architecture == native ELF architecture...
*** package/uclibc/uclibc.mk.orig	2014-03-12 10:24:01.049045437 -0700
--- package/uclibc/uclibc.mk	2014-03-14 09:59:58.161737212 -0700
***************
*** 525,534 ****
  # For FLAT binfmts (static) there are no host utils
  ifeq ($(BR2_BINFMT_FLAT),)
  define UCLIBC_INSTALL_UTILS_STAGING
! 	$(INSTALL) -D -m 0755 $(@D)/utils/ldd.host $(HOST_DIR)/usr/bin/ldd
! 	ln -sf ldd $(HOST_DIR)/usr/bin/$(GNU_TARGET_NAME)-ldd
! 	$(INSTALL) -D -m 0755 $(@D)/utils/ldconfig.host $(HOST_DIR)/usr/bin/ldconfig
! 	ln -sf ldconfig $(HOST_DIR)/usr/bin/$(GNU_TARGET_NAME)-ldconfig
  endef
  endif
  
--- 526,538 ----
  # For FLAT binfmts (static) there are no host utils
  ifeq ($(BR2_BINFMT_FLAT),)
  define UCLIBC_INSTALL_UTILS_STAGING
! 	#Till Straumann: do NOT install as 'ldd/ldconfig' since this may
! 	#result in this ldd being used when the native/host ldd is wanted.
! 	#Fixes a problem when configuring ncurses; uclibc's ldd produces
! 	#output which is incompatible with ncurses-5.7 'configure'
! 
! 	$(INSTALL) -D -m 0755 $(@D)/utils/ldd.host $(HOST_DIR)/usr/bin/$(GNU_TARGET_NAME)-ldd
! 	$(INSTALL) -D -m 0755 $(@D)/utils/ldconfig.host $(HOST_DIR)/usr/bin/$(GNU_TARGET_NAME)-ldconfig
  endef
  endif
  
***************
*** 547,553 ****
  		$(UCLIBC_MAKE_FLAGS) \
  		PREFIX=$(STAGING_DIR) \
  		DEVEL_PREFIX=/usr/ \
! 		RUNTIME_PREFIX=$(STAGING_DIR)/ \
  		menuconfig
  	rm -f $(UCLIBC_DIR)/.stamp_{configured,built,target_installed,staging_installed}
  
--- 551,557 ----
  		$(UCLIBC_MAKE_FLAGS) \
  		PREFIX=$(STAGING_DIR) \
  		DEVEL_PREFIX=/usr/ \
! 		RUNTIME_PREFIX=$(TARGET_DIR)/ \
  		menuconfig
  	rm -f $(UCLIBC_DIR)/.stamp_{configured,built,target_installed,staging_installed}
  
